pkg/ocm/clusters.go

```diff
@@ -604,3 +639,44 @@ func (c *Client) ResumeCluster(clusterID string) error {
 
 	return nil
 }
+
+//Get all the archived ROSA STS clusters
+func (c *Client) GetArchivedCluster(clusterKey string) (clusters []*cmv1.Cluster, err error) {
+	query := fmt.Sprintf("id = '%s' OR name = '%s' OR external_id = '%s'",
+		clusterKey, clusterKey, clusterKey,
+	)
```

Comment:
This should also filter by `rosa` and clusters in the current account, to be consistent:
```
	query := fmt.Sprintf("%s AND (id = '%s' OR name = '%s' OR external_id = '%s')",
		getClusterFilter(creator),
		clusterKey, clusterKey, clusterKey,
	)
```